src/App.tsx

```diff
@@ -35,6 +39,20 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
+  switch (sortType) {
+    case SortType.ALPABET:
+      visibleGoods.sort((goodA, goodB) => goodA.localeCompare(goodB));
+      break;
+    case SortType.LENGTH:
+      visibleGoods.sort((goodA, goodB) => goodA.length - goodB.length);
+      break;
+    default:
```

Comment:
default value should return 0

Reply:
I like your solution to use a switch operator inside the callback. So I made changes to apply it to my code.